extensions/ql-vscode/src/databases/git-hub-url-identifier-helper.ts

```diff
@@ -0,0 +1,47 @@
+import { Uri } from "vscode";
+import { OWNER_REGEX, REPO_REGEX } from "../pure/helpers-pure";
+
+/**
+ * Checks if a string is a valid GitHub owner or NWO.
+ * @param identifier The GitHub owner or NWO
+ * @param onlyOwner If true, validate an owner, otherwise a repository
+ * @returns
+ */
+export function validGitHubNwoOrOwner(
```

Comment:
[Also discussed on Slack]

We can split this up into 2 separate functions for owner and nwo ⚡

Reply:
👍 did the same for getNwoOrOwner